_posts/en/newsletters/2024-01-31-newsletter.md

```diff
@@ -0,0 +1,253 @@
+  package providing a common interface to multiple different hardware
+  signing devices.
+
+<!-- FIXME:harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [Bitcoin Core #29291][] adds a test that will fail if a transaction
+  executing an `OP_CHECKSEQUENCEVERIFY` opcode appears to have a
+  negative version number.  This test, if it had been run by alternative
+  consensus implementations, would have caught the consensus failure bug
+  mentioned in [last week's newsletter][news286 bip68ver].
+
+- [Eclair #2811][], [#2813][eclair #2813], and [#2814][eclair #2814]
+  adds the ability for a [trampoline payment][topic trampoline payments]
+  to use a [blinded path][topic rv routing] for the ultimate receiver.
+  Trampoline routing itself continues to use regular onion-encrypted
+  node IDs, i.e. each trampoline node learns the ID for the next
+  trampoline node.  However, if a blinded path is used, the final
+  trampoline node will now only learn the node ID for the first
+  forwarding node in the blinded path; it will not learn the ID for
+  the ultimate receiver.
+
+    Previously, strong trampoline privacy depended on using multiple
+    trampoline forwarders so that none of the forwarders could be sure
+    they were the final forwarder.  A downside of this approach is that
+    it used longer paths that increased the probability of forwarding
+    failure and required paying more forwarding fees for success.  Now
+    forwarding payments through even a single trampoline node can
+    prevent that node from learning the ultimate receiver.
+
+- [LND #8167][] allows an LND channel to mutually close a channel that
+  still has one or more pending payments ([HTLCs][topic htlc]).  The
+  [BOLT2][] specification indicates the proper procedure for this is for
+  one side to send a `shutdown` message, after which no new HTLCs will
+  be accepted.  After all pending HTLCs are resolved offchain, the two
+  parties negotiate and sign a mutual close transaction.  Previously,
+  when LND received a `shutdown` message, it would force close the
+  channel, requiring extra onchain transactions and fees to settle.
+
+- [LND #7733][] updates [watchtower][topic watchtowers] support to enable
+  backing up and enforcing correct shutdown of the [simple taproot
+  channels][topic simple taproot channels] that are now supported
+  experimentally by LND.
+
+- [LND #8275][] begins requiring peers support certain
```

Comment:
```suggestion
- [LND #8275][] begins requiring peers to support certain
```

Reply:
as-is still sounds off to me -- as a different suggestion, maybe "begins requiring that peers support certain"
(this is minor)